common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -4,6 +4,9 @@ info:
   description: |
     De definitie van generieke datatypen die worden gebruikt door gemeentelijke API's.
     Deze datatypen komen overeen met het basismodel Generieke Datatype Gemeenten waarnaar verwezen wordt vanuit informatiemodellen.
+    Bij het opstellen van de JSON schema-componenten worden de naamgevingsconventies van het BSM toegepast.
+    Er zijn dus (verklaarbare) verschillen tussen de namen in het basismodel en dit Json Schema.
+
```

Comment:
Ik zou er wel voor willen pleiten dat de namen van de datatypen zelf wel gelijk zijn aan de namen van de componenten in dit schema. Dat zorgt er voor dat het algoritme dat de referentie naar dat component genereert heel eenvoudig het path op kan halen uit een centrale bron en daar de naam van het datatype aan kan toevoegen.

Reply:
Begrijp ik, dan moeten we dat dus opnemen in de beschrijving van het model "Generieke Datatype Gemeenten" . 

Dat betekent dat bij het definieren van een generiek datatype hier rekening mee gehouden wordt. Ik opteer er ook voor om voorlopig alleen door jou en mij datatypen toe te laten voegen. Pas als iemand echt begrijpt wat hij doet laten we dit ook door anderen doen.